R/utils.R

```diff
@@ -43,3 +43,22 @@ to_datetime_origin <- function(x) {
 trim_ids_list <- function(ids_list, max_items) {
   ids_list[1:min(max_items, length(ids_list))]
 }
+
+#' Converts comment to a dataframe row
+#'
+#' @param comment comment to be converted to dataframe
+#'
+#' @return a dataframe containing a single entry with data from passed comment
+#'
+comment_to_dataframe_row <- function(comment){
+  data.frame(
+    id = comment[['id']],
+    deleted = ifelse(is.null(comment[['deleted']]), FALSE, comment[['deleted']]),
+    by =  ifelse(is.null(comment[['by']]), '', comment[['by']]),
+    time = comment[['time']],
+    text = ifelse(is.null(comment[['text']]), '', comment[['text']]),
+    dead = ifelse(is.null(comment[['dead']]), FALSE, comment[['dead']]),
+    parent = comment[['parent']],
+    stringsAsFactors = FALSE
+  )
+}
```

Comment:
- Are you sure that those are all of the fields that can appear in a comment?
- Please use NA instead of empty strings ;P 
- I am also a bigger fan of using `comment$id` and `comment[['id']]` maybe let's keep a common convention in the package and use the `$` :)

Reply:
Yes, I'm sure. I checked it in table at https://github.com/HackerNews/API. Rest is fixed.